### PR TITLE
[feat]: Quiz page

### PR DESCRIPTION
**Pull request by Ananyamadhu08**
- created quiz page

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  opacity: 0;
+}
+.search__bar:focus:-ms-placeholder {
+  transition: opacity 0.45s ease;
+  opacity: 0;
+}
+
+.search__icon {
+  position: relative;
+  left: -30px;
```
Comment (abhishektanwar) [inline index.css:41]: use Rems here?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        </div>
+        <!-- header right (nav-links, login/signup CTA) -->
+      </div>
+    </header>
+    <!-- header -->
+
+    <!-- main section -->
+    <main
+      class="relative w-full flex align-items-center justify-center"
+      style="top: 5rem; min-height: 100vh"
```
Comment (abhishektanwar) [inline index.html:99]: Please try to avoid inline styling.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    >
+      <div style="max-width: 980px; margin: auto">
+        <div class="flex flex-col justify-center mb-10">
+          <h2 class="mb-10 text-8xl text-cyan-600">Quiz Mania</h2>
+
+          <h5 class="text-xl">
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Assumenda,
+            <br />
```
Comment (abhishektanwar) [inline index.html:107]: why are we using break here ? You can use ch unit with container width to restrict according to number of characters.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+        </a>
+        <a href="https://twitter.com/AnanyaMadhu27" target="_blank">
+          <i class="fab fa-twitter mr-2 text-cyan-700"></i>
+        </a>
+        <a
+          href="https://www.linkedin.com/in/ananya-madhu-74479b206/"
+          target="_blank"
+        >
+          <i class="fab fa-linkedin mr-2 text-cyan-700"></i>
+        </a>
```
Comment (abhishektanwar) [inline index.html:151]: Using rel="noopener noreferrer" is a good practice from security point

File: pages/quiz.html [hunk 5]
```diff
@@ -0,0 +1,333 @@
...
+            </div>
+          </div>
+
+          <div
+            class="shadow-lg rounded flex flex-col relative w-72 mb-12 ml-12 mr-5"
+          >
+            <div class="h-72">
+              <img
+                class="img-responsive rounded"
+                style="height: 100%"
```
Comment (abhishektanwar) [inline pages/quiz.html:223]: avoid inline styling

File: pages/quiz.html [hunk 6]
```diff
@@ -0,0 +1,333 @@
...
+        </a>
+        <a href="https://twitter.com/AnanyaMadhu27" target="_blank">
+          <i class="fab fa-twitter mr-2 text-cyan-700"></i>
+        </a>
+        <a
+          href="https://www.linkedin.com/in/ananya-madhu-74479b206/"
+          target="_blank"
+        >
+          <i class="fab fa-linkedin mr-2 text-cyan-700"></i>
+        </a>
```
Comment (abhishektanwar) [inline pages/quiz.html:299]: It is advised to use rel="noopener noreferrer"  with target="_blank" to improve security.

File: pages/quiz.html [hunk 7]
```diff
@@ -0,0 +1,333 @@
...
+      closeBtn.onclick = function () {
+        modal.style.display = "none";
+      };
+
+      window.onclick = function (event) {
+        if (event.target == modal) {
+          modal.style.display = "none";
+        }
+      };
+    </script>
```
Comment (abhishektanwar) [inline pages/quiz.html:331]: Question : Why are we writing JS in HTML file . Could we make a separate file for this ?
Also using const or let would be much better. 
Remove commented code

**Review by VedanthB — APPROVED** [review]
Good work ananya, please try to attach screen shots or a preview link,  it helps with the reviewing.

**Review by abhishektanwar — CHANGES_REQUESTED** [review]
Please check comments . We are writing JS in HTML file .It would be better if you could write logic in a separate JS file.

**Pull request merged**